Tasks/DownloadBuildArtifacts/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,20 +1,31 @@
 {
-  "loc.friendlyName": "Download Artifacts - Build",
+  "loc.friendlyName": "Download Build Artifacts",
   "loc.helpMarkDown": "",
-  "loc.description": "Download build artifacts",
-  "loc.instanceNameFormat": "Download Artifacts - Build",
+  "loc.description": "Download Build Artifacts",
+  "loc.instanceNameFormat": "Download Build Artifacts",
+  "loc.input.label.buildType": "Download artifacts produced by",
+  "loc.input.help.buildType": "Download artifacts produced by the current build, or from a specific build.",
   "loc.input.label.project": "Project",
-  "loc.input.help.project": "Select the project name",
+  "loc.input.help.project": "The project from which to download the build artifacts",
   "loc.input.label.definition": "Build definition",
   "loc.input.help.definition": "Select the build definition name",
-  "loc.input.label.version": "Build",
-  "loc.input.help.version": "Select the build number",
-  "loc.input.label.itemPattern": "Item Pattern",
-  "loc.input.help.itemPattern": "Minmatch pattern to filter files to be downloaded.",
+  "loc.input.label.buildId": "Build",
+  "loc.input.help.buildId": "The build from which to download the artifacts",
+  "loc.input.label.downloadType": "Download Type",
+  "loc.input.help.downloadType": "Download specific or multiple artifacts matching pattern.",
```

Comment:
I'm not sure this one makes sense.  It implies that the artifact has to match the pattern, when from the code, and the description of itemPattern below, it appears that it refers to specific files within the artifact.

Reply:
@scottdallamura for the itemPattern input we have enhanced the text by an example of how to specify the pattern on PM feedback. I will remove "matching pattern" for here so as not to create confusion.